misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/LocalDateTimeFeatureAssertionsSpec.kt

```diff
@@ -41,23 +45,28 @@ abstract class LocalDateTimeFeatureAssertionsSpec(
         monthFeature.forSubjectLess().adjustName { "$it feature" },
         month.forSubjectLess { isLessThan(12) },
         dayOfWeekFeature.forSubjectLess().adjustName { "$it feature" },
-        dayOfWeek.forSubjectLess { isLessOrEquals(DayOfWeek.SUNDAY) }
+        dayOfWeek.forSubjectLess { isLessOrEquals(DayOfWeek.SUNDAY) },
+        dayFeature.forSubjectLess().adjustName { "$it feature" },
+        day.forSubjectLess { isLessOrEquals(20) }
     ) {})
 
     include(object : AssertionCreatorSpec<LocalDateTime>(
-        describePrefix, LocalDateTime.now().withYear(2040).withDayOfYear(1),
+        describePrefix, LocalDateTime.now().withYear(2040).withDayOfYear(1).withDayOfMonth(15),
         year.forAssertionCreatorSpec("$toBeDescr: 1") { toBe(2040) },
         month.forAssertionCreatorSpec("$toBeDescr: 1") {toBe(1)},
-        dayOfWeek.forAssertionCreatorSpec("$toBeDescr: 1") {toBe(DayOfWeek.SUNDAY)}
+        dayOfWeek.forAssertionCreatorSpec("$toBeDescr: 1") {toBe(DayOfWeek.SUNDAY)},
+        day.forAssertionCreatorSpec("$toBeDescr: 1") {toBe(15)}
+
     ) {})
 
     fun describeFun(vararg funName: String, body: Suite.() -> Unit) =
         describeFunTemplate(describePrefix, funName, body = body)
 
-    val fluent = expect(LocalDateTime.now().withMonth(5).withYear(2009).withDayOfMonth(15))
+    val fluent = expect(LocalDateTime.of(2009,5,15,10,5))
```

Comment:
I changed this because in expect message time was always changing to current time. And i was unable to assert the message. So i made it static.

Reply:
That's a good decision :+1: